src/nrc/api/tasks.py

```diff
@@ -1,32 +1,79 @@
 import json
 import logging
 
-from django.conf import settings
 from django.core.management import call_command
 from django.core.serializers.json import DjangoJSONEncoder
 from django.utils.translation import gettext_lazy as _
 
 import requests
+from celery.exceptions import Ignore, Retry
+from celery.utils.time import get_exponential_backoff_interval
+from notifications_api_common.models import NotificationsConfig
+from vine.utils import wraps
 
 from nrc.celery import app
 from nrc.datamodel.models import Abonnement, NotificatieResponse
 
 logger = logging.getLogger(__name__)
 
 
+def add_autoretry_behaviour(task, **options):
```

Comment:
Can't you reuse function from `notifications-api-common`? It looks like it does the same thing

Reply:
Ah you're right, replaced it